projects/plugins/social/tests/e2e/specs/social-sidebar.test.js

```diff
@@ -0,0 +1,51 @@
+import { prerequisitesBuilder } from 'jetpack-e2e-commons/env/prerequisites.js';
+import { expect, test } from 'jetpack-e2e-commons/fixtures/base-test.js';
+import logger from 'jetpack-e2e-commons/logger.js';
+import BlockEditorPage from 'jetpack-e2e-commons/pages/wp-admin/block-editor.js';
+import { connect } from '../flows/index.js';
+
+test.beforeEach( async ( { page } ) => {
+	await prerequisitesBuilder( page )
+		.withCleanEnv()
+		.withActivePlugins( [ 'social' ] )
+		.withInactivePlugins( [ 'jetpack' ] )
+		.withLoggedIn( true )
+		.withWpComLoggedIn( true )
```

Comment:
For some reason this step is faliing for me. It gets stuck on the confirmation screen after login. It must be something about my environment or other tests would be failing.

Is it still working for you in your dev environment?

Reply:
Sorry meant to attach the screenshot

![test-failed-1](https://github.com/Automattic/jetpack/assets/96462/c79f2342-798c-446e-a986-32cc4e71986d)

Reply:
> Is it still working for you in your dev environment?

Yes.

What error do you see in the console/terminal?

Reply:
It's a timeout on this page as it's waiting for the form to be hidden, but that never happens as we need to click on the continue button. Very strange! The GH Action is succeeding too. I've tried cleaning my environment but I get the same problem.

Reply:
Yeah, many of those workflow utilities are unpredictable and needs more work, which is beyond the scope of this PR.

Reply:
I've pushed a change in f670105, that looks for the Login button being hidden rather than waiting for the login container to be hidden (which isn't happening sometimes). It works locally for me, but we'll have to see if it works in CI and for everyone else.

Reply:
I don't see anything changed in that commit you pushed 🤔

Reply:
Yeah! I've committed the wrong file. 49f6911 should have the right changes.